cmd/copy.go

```diff
@@ -376,16 +376,16 @@ func (raw rawCopyCmdArgs) cook() (cookedCopyCmdArgs, error) {
 		}
 		// Disabling blob tier override, when copying block -> block blob or page -> page blob, blob tier will be kept,
 		// For s3 and file, only hot block blob tier is supported.
+		if cooked.fromTo == common.EFromTo.FileBlob() && cooked.blobType != common.EBlobType.None() {
+			return cooked, fmt.Errorf("file -> blob is not a supported direction while copying from service to service")
+		}
+		if cooked.fromTo == common.EFromTo.S3Blob() && cooked.blobType != common.EBlobType.None() {
+			return cooked, fmt.Errorf("custom blob type is not supported while copying from s3 to blob")
```

Comment:
>custom blob type [](start = 30, length = 16)

Our convention s to use the actual parameter names in messages like this.  So the message would be "blob-type is not supported while..." (instead of using the phrase "custom blob type" to mean the blob-type param)